server/src/main/java/uplink/db/CommandController.java

```diff
@@ -0,0 +1,55 @@
+package uplink.db;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.bson.Document;
+import org.bson.types.ObjectId;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.mongodb.MongoClient;
+import com.mongodb.MongoClientURI;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+
+import models.User;
+
+@RestController
+public class CommandController {
+
+	MongoClientURI uri = new MongoClientURI(APIKeys.getMongoDBPassword());
```

Comment:
Do you mind sending over the code for the APIKeys class? Just so everyone can have the same structure.

Reply:
Just sent it to the Slack channel.